examples/Services/Recommendations/DismissRecommendation.cs

```diff
@@ -0,0 +1,100 @@
+using static Google.Ads.GoogleAds.V0.Services.DismissRecommendationResponse.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V0
+{
+    /// <summary>
+    /// This code example dismisses a given recommendation. To retrieve recommendations for text
+    /// ads, run GetTextAdRecommendations.cs.
+    /// </summary>
+    public class DismissRecommendation : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            DismissRecommendation codeExample = new DismissRecommendation();
+            Console.WriteLine(codeExample.Description);
+
+            //The customer ID for which the call is made.
+            int customerId = int.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // ID of the recommendation to be dismissed.
+            int recommendationId = int.Parse("INSERT_RECOMMENDATION_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId, recommendationId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example dismisses a given recommendation. To retrieve " +
+                    "recommendations for text ads, run GetTextAdRecommendations.cs.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The customer ID for which the call is made.</param>
+        /// <param name="recommendationId">ID of the recommendation to dismiss.</param>
+        public void Run(GoogleAdsClient client, long customerId, long recommendationId)
+        {
+            // Get the RecommendationServiceClient.
+            RecommendationServiceClient service = client.GetService(
+                Services.V0.RecommendationService);
+
+            DismissRecommendationOperation operation = new DismissRecommendationOperation()
+            {
+                ResourceName = ResourceNames.Recommendation(customerId, recommendationId),
+            };
+
+            bool partialFailure = false;
+
+            try
+            {
+                DismissRecommendationResponse response = service.DismissRecommendation(
+                    customerId.ToString(), partialFailure, new DismissRecommendationOperation[] {
+                        operation
+                    });
+                Console.WriteLine($"Dismissed {0} recommendation(s):", response.Results.Count);
+                foreach (DismissRecommendationResult result in response.Results)
+                {
+                    Console.WriteLine($"- {result.ResourceName}");
```

Comment:
Is this how you're printing lists in other examples? Would be good to be consistent.

Reply:
Done